src/dynaint.rs

```diff
@@ -886,6 +887,19 @@ where
     dyna_impl!(impl_fn_refmath_tuple2self; div_rem);
 }
 
+impl<T, G> ToBigInt for DynaInt<T, G>
```

Comment:
Thanks. I've added this on `ToBigInt` and a new `ToBigUint` as well as for the whole `approx` module (for now; see [my comment below](https://github.com/dnsl48/fraction/pull/84#discussion_r1314323501)).

Reply:
I've now changed things so that the `sqrt_*` methods directly return `GenericFraction<BigUint>` (rather than `DynaFraction`, which was what it was before). This is because
1. Any reasonable approximation will have a numerator and denominator so large that it would not be possible to store them even in `u128`, making `DynaFraction` essentially equivalent to `GenericFraction<BigUint>` anyway;
2. The algorithm itself works with `BigUint`s, so returning a fraction made of `BigUint`s means there's one less processing step before output; and
3. If the user wants a `DynaFraction`, then they can do that for themselves.

However, I've left the implementations of `ToBigInt` and `ToBigUint` for `DynaInt` because they allow `sqrt` for `DynaFraction`.

Reply:
that sounds good, thank you!